communitiesuk/funding-service-pre-award
# Send "application deadline reminder" email synchronously

https://mhclgdigital.atlassian.net/browse/FSPT-172

### Change description
This changes the emails sent when the "application deadline reminder" task/script is executed to happen via synchronous calls to GOV.UK Notify, rather than SQS messages picked up by the separate notifications service.



### How to test
1. Start an application for the Crash Test Dummy Fund
2. Edit the `reminder_date` for that fund in your DB to be in the past
3. Exec into the pre-award container: `docker exec -it funding-service-pre-award-1 bash`
4. Open a flask shell: `flask shell`
5. Run `from application_store.scripts.send_application_reminder import *; application_deadline_reminder(app)`
6. Log into GOV.UK Notify and check the web API console to confirm the email was sent to you.


### Screenshots of UI changes (if applicable)
